src/main/java/graphql/schema/idl/TypeDefinitionRegistry.java

```diff
@@ -636,11 +731,11 @@ public List<ImplementingTypeDefinition> getAllImplementationsOf(InterfaceTypeDef
      * @see TypeDefinitionRegistry#getAllImplementationsOf(InterfaceTypeDefinition)
      */
     public List<ObjectTypeDefinition> getImplementationsOf(InterfaceTypeDefinition targetInterface) {
-        return this.getAllImplementationsOf(targetInterface)
-                .stream()
-                .filter(typeDefinition -> typeDefinition instanceof ObjectTypeDefinition)
-                .map(typeDefinition -> (ObjectTypeDefinition) typeDefinition)
-                .collect(Collectors.toList());
+        return ImmutableKit.filterAndMap(
+                getAllImplementationsOf(targetInterface),
+                typeDefinition -> typeDefinition instanceof ObjectTypeDefinition,
+                typeDefinition -> (ObjectTypeDefinition) typeDefinition
+        );
```

Comment:
unwound the `.stream()` and replaced it with a cheaper immutable builder version.  This is known to be faster